e2e/chatqna/README.md

```diff
@@ -0,0 +1,27 @@
+
+## Prerequisites
+* Provisioned RHOCP cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster)
+* Provisioning Intel Gaudi 2 accelerator:
+  * Setup KMM and node feature discovery (NFD) for drivers. Follow the steps [here](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Environment_Setup.html)
+  * Setup HabanaAI operator and create device config custom Resource. Follow the steps [here](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/Deploying_HabanaAI_Operator.html)
+* Install RHOAI. Follow steps [here](../inference/README.md/#install-rhoai)
+
+## Deploy workload
+* The workload uses docker-compose yaml's. They can be converted into OCP compatible yaml's using [Kompose tool](https://kubernetes.io/docs/tasks/configure-pod-container/translate-compose-kubernetes/). This creates the corresponding deployment and service yaml's.
+
+### ChatQnA backend- langchain and redis vector database
+* Please follow the [docker-compose file](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/deprecated/langchain/docker/docker-compose.yml). To create the backend image, use the [dockerfile](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/deprecated/langchain/docker/Dockerfile). Make sure to mount the langchain folder while building image and deploying the pod. The redis vector database image used is ``` redis/redis-stack:7.2.0-v9 ```. Please update the environment variables according to your setup and model variables in the yaml's.
+
+### Minio storage for models
+Minio/S3 storage format is supported on RHOAI. To setup minio on OCP, please follow the steps in [documentation](https://ai-on-openshift.io/tools-and-applications/minio/minio/#create-a-matching-data-connection-for-minio). Make sure to allocate enough storage based on the model weights. Once the minio-ui can be accessed, create the bucket and add your model files. Also create an access key and secret key.
+
+### Text Generation Inference for Gaudi on RHOAI
+1. Access rhods-dashboard link from routes on your web console and click on Data Science Projects. Now, create a project.
+2. Click on the project, and choose ```data connection``` section. In the fields, add your access and secret keys from minio, along with minio-api route (from your minio deployment in previous step). This sets up the connection for your model to be downloaded and prepare for serving.
```

Comment:
Do we have to do it manually? Do we have a yaml file to do it automatically?

Reply:
This is on the web console of rhoai, we have to add it there. no yaml's